src/foam/nanos/auth/twofactor/GoogleTOTPAuthService.js

```diff
@@ -46,14 +47,16 @@ foam.CLASS({
 
   methods: [
     {
-      name: 'generateKey',
+      name: 'generateKeyAndQR',
       javaCode: `
+        Logger logger = (Logger) x.get("logger");
```

Comment:
I think this is no longer needed. Unless you also want to log errors so we can monitor how frequently we fail to fetch QR codes.